File: PHP/Iniciante/1008.php
<?php

$n_func = intval(readline());
$n_horas = intval(readline());
$sal_hora = floatval(readline());

$salario = number_format($n_horas * $sal_hora, 2, '.', '');

echo "NUMBER = {$n_func}\n";
echo "SALARY = U$ {$salario}\n";

?>

File: C#/Iniciante/1008.cs
using System; 

class URI {

    static void Main(string[] args) { 

        int n = Convert.ToInt32(Console.ReadLine());
        int n_horas = Convert.ToInt32(Console.ReadLine());
        double valor_hora = Convert.ToDouble(Console.ReadLine());
        
        double salario = n_horas * valor_hora;
        
        Console.WriteLine($"NUMBER = {n}");
        Console.WriteLine($"SALARY = U$ {salario:F2}");

    }

}
